feature: Document ignore security findings feature [TAROT-2883] (#2240)
codacy/docs: docs/organizations/managing-security-and-risk.md

!!! Important " [**Segments**](../segments) filter won't be considered when sharing the filtered view"

## Ignoring findings {: id="ignoring-findings"}

!!! info "This feature is available only to organization admins and organization managers"

> In the findings details page, it's possible to ignore it using the context menu. When ignoring an issue you can optionally specify a reason for doing so.

Reviewer: ```suggestion
On the finding's details page, you can ignore a finding using the context menu. When ignoring a finding, you can optionally specify a reason for doing so.
```

![Security and risk management finding ignore](images/security-risk-management-finding-ignore.png)

From an organization standpoint, ignoring a finding means that you accept the risk it poses and you're not planning on addressing the issue.

From Codacy's standpoint, ignoring a finding means it will be removed from the metrics featured in the [overview page](#dashboard) page. Note that the [Open Findings history](#open-findings-history) chart will only be changed at the start of next week.

!!! info [Jira](./integrations/jira-integration.md) findings can't be ignored in Codacy. You should closed the issue directly in Jira.

!!! warn Ignoring findings detected on [Git repositories](#opening-and-closing-items) will also [ignore the issue at the repository level](../repositories/issues.md#ignoring-and-managing-issues).

You can still see **Ignored** findings in the [findings list](#item-list), by filtering for the **Ignored** status in the **Status** dropdown. Check the **Status** column to know the status of a finding.

![Security and risk management finding unignore list](images/security-risk-management-finding-unignore-list.png)

> An Ignored finding can be **unignored** directly from the [findings list](#item-list) or by going to the same menu where the ignore action was performed, in the findings details page. Note that in this page you can also find out more about who ignored the issue and why, if such a reason was provided.

Reviewer: ```suggestion
An Ignored finding can be **unignored** directly from the [findings list](#item-list) or by going to the same menu where the ignore action was performed, in the findings details page. Note that in this page you can also find out more about who ignored the finding and why, if such a reason was provided.
```
